odoo/addons/base/tests/test_res_lang.py

```diff
@@ -40,3 +42,63 @@ def test_00_intersperse(self):
         assert intersperse("abc12", [3], '.') == ('abc12', 0)
         assert intersperse("abc12", [2], '.') == ('abc12', 0)
         assert intersperse("abc12", [1], '.') == ('abc1.2', 1)
+
+    def test_res_lang_decimal_point_preview(self):
+        lang_0 = self.env['res.lang'].create({
```

Comment:
You could add a setupClass method to create a test_lang record.

That way we gain one creation as it's used in multiple tests.